modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -467,3 +471,65 @@ function editConsentStatusFields($db, $user)
 
     }
 }
+
+/**
+ * Handles the updating of External Identifier
+ *
+ * @param Database $db   database object
+ * @param User     $user user object
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editExternalIdentifierFields($db, $user)
+{
+    if (!$user->hasPermission('candidate_parameter_edit')) {
+        header("HTTP/1.1 403 Forbidden");
+        exit;
```

Comment:
```suggestion
        return;
```
Dave gets mad if you use hard exits.